### PR TITLE
Add `GetUser` method

### PR DESCRIPTION
**Pull request by pliao28**
## Description
Add UserManagement Service
Add User Entitiy
Add GetUser Function

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: .semaphore/publish.yml [hunk 1]
```diff
@@ -1,12 +1,9 @@
 version: v1.0
-
```
Comment (maxdeviant) [inline .semaphore/publish.yml:2]: Are these changes intentional?

File: src/WorkOS.net/Services/UserManagement/UserManagementService.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        {
+            var request = new WorkOSRequest
+            {
+                Method = HttpMethod.Get,
+                Path = $"/users/{id}",
+            };
+            return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
+        }
+    }
+}
```
Comment (maxdeviant) [inline src/WorkOS.net/Services/UserManagement/UserManagementService.cs:48]: Could we make sure all files have a trailing newline?

```suggestion
}

```

File: .semaphore/publish.yml (hunk 1 above)
Comment (maxdeviant) [reply .semaphore/publish.yml:2]: Was the configuration done in the Semaphore web UI?

We should make the changes in the YAML file directly to avoid unnecessary changes.

File: src/WorkOS.net/WorkOS.net.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>net5.0;net6.0;net7.0;netcoreapp3.1;netstandard2.0;net461</TargetFrameworks>
```
Comment (maxdeviant) [inline src/WorkOS.net/WorkOS.net.csproj:3]: Could we split the changes to the compilation targets out into a separate PR?

Comment (pliao28) [reply src/WorkOS.net/WorkOS.net.csproj:3]: Moved to https://github.com/workos/workos-dotnet/pull/134

**Comment by pliao28** [comment]
@maxdeviant Reverted yaml and framework targeting

File: src/WorkOS.net/Services/UserManagement/Entities/User.cs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        /// The timestamp when the user's email was verified. Email verification is only applicable to unmanaged users.
+        /// </summary>
+        [JsonProperty("email_verified_at")]
+        public string EmailVerifiedAt { get; set; }
+
+        /// <summary>
+        /// The ID of the Google OAuth Profile. Only unmanaged users who sign in with Google OAuth have Google OAuth Profiles.
+        /// </summary>
+        [JsonProperty("google_oauth_profile_id")]
+        public string GoogleOAuthProfileId { get; set; }
```
Comment (maxdeviant) [inline src/WorkOS.net/Services/UserManagement/Entities/User.cs:58]:
```suggestion
        public string GoogleOauthProfileId { get; set; }
```

File: src/WorkOS.net/Services/UserManagement/Entities/User.cs [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+        /// <summary>
+        /// The ID of the SSO Profile. Only managed users have SSO Profiles.
+        /// </summary>
+        [JsonProperty("sso_profile_id")]
+        public string SSOProfileId { get; set; }
```
Comment (maxdeviant) [inline src/WorkOS.net/Services/UserManagement/Entities/User.cs:64]:
```suggestion
        public string SsoProfileId { get; set; }
```

**Review by maxdeviant — APPROVED** [review]
